### PR TITLE
fix: update the expected responses placeholder

### LINKED ISSUE #7816: In a RegEx Trigger the text for the expression is inconsistent in creation and editing flows
## Describe the bug

When you create a new RegEx Trigger the dialog that comes up says "Please input regEx pattern" once it's created the UX says "Trigger phrases" for the same value, these should be consistent (preferably both containing RegEx) 

## Version

2.0.0-nightly.245007.8406071

## Browser

Stand alone windows app

## OS

- [ ] macOS
- [ X] Windows
- [ ] Ubuntu

## To Reproduce

1. Go to initial dialog
2. make it a regex bot
3. Click on Add new trigger
4. Add the new trigger, observe dialog text during creation
5. Now look at created page for trigger and observer

## Expected behavior

consistency and clarity in naming

## Screenshots

CreateATrigger
EditATrigger

### PR DESCRIPTION
**Pull request by lei9444**
## Description
The expected responses' placeholder is the same under different recognizers. The placeholder of regex recognizer is like
image
this may confuse users.

**fix**
update the placeholder for regex recognizer

## Task Item
refs #7779
fixes #7816 

## Screenshots
placehode

### CODE REVIEW

File: Composer/packages/ui-plugins/prompts/src/ExpectedResponsesField.tsx [hunk 1]
```diff
@@ -3,19 +3,32 @@
...
+      > - Please remind me to '{itemTitle=buy milk}'
+      > - remind me to '{itemTitle}'
+      > - add '{itemTitle}' to my todo list
+      >
+      > entity definitions:
+      > @ ml itemTitle
+      `);
+      break;
+    case SDKKinds.RegexRecognizer:
+      placehold = formatMessage('ex. Book a flight to(?<toCity>.*)');
```
Comment (boydc2014) [inline Composer/packages/ui-plugins/prompts/src/ExpectedResponsesField.tsx:25]: Let's probably call this, "Add some regex patterns"

File: Composer/packages/ui-plugins/prompts/src/ExpectedResponsesField.tsx [hunk 2]
```diff
@@ -3,19 +3,32 @@
...
+      > - Please remind me to '{itemTitle=buy milk}'
+      > - remind me to '{itemTitle}'
+      > - add '{itemTitle}' to my todo list
+      >
+      > entity definitions:
+      > @ ml itemTitle
+      `);
+      break;
+    case SDKKinds.RegexRecognizer:
+      placehold = formatMessage('Add some regex patterns');
```
Comment (tdurnford) [inline Composer/packages/ui-plugins/prompts/src/ExpectedResponsesField.tsx:25]: We should change this to ‘Add regex pattern’.

Comment (hatpick) [reply Composer/packages/ui-plugins/prompts/src/ExpectedResponsesField.tsx:25]: nit: please name the variable ```placeholderText```

**Review by boydc2014 — APPROVED** [review]

File: Composer/packages/ui-plugins/prompts/src/ExpectedResponsesField.tsx [hunk 3]
```diff
@@ -3,19 +3,32 @@
...
->
-> entity definitions:
-> @ ml itemTitle
-`);
+const expectedResponsesPlaceholder = (id?: SDKKinds) => {
+  let placeholderText = '';
+  switch (id) {
+    case SDKKinds.CrossTrainedRecognizerSet:
+    case SDKKinds.LuisRecognizer:
+      placeholderText = formatMessage(`> add some expected user responses:
```
Comment (hatpick) [inline Composer/packages/ui-plugins/prompts/src/ExpectedResponsesField.tsx:15]: @lei9444 these spaces you have added to the beginning of lines 16-22 (inclusive) will get rendered in monaco, please fix.

Screen Shot 2021-07-08 at 11 06 51 AM

**Pull request merged**